### ISSUE TITLE
Pretrained weights.  Validation accuracy.

### ISSUE THREAD

**Issue by tempdeltavalue** [fmahoudeau/MiCT-Net-PyTorch#5]
Hi, sorry to bother you but can you share pre-trained weights? 
Also I have a question accuracy which is you presented in in readme this is validation or train accuracy ?

**Comment by fmahoudeau** [comment]
Hi, I have uploaded pre-trained weights to OneDrive. You can find the links in the README.
All accuracies reported are cross-validated accuracies using the 3 official splits of UCF-101.
You can find an example learning curve [here](https://github.com/fmahoudeau/MiCT-Net-PyTorch/blob/master/results/MiCT-ResNet-18-7x7x7.jpg)
The best result so far is achieved on split-1 of MiCT-ResNet-34 with 73.8 top1 acc.

**Comment by tempdeltavalue** [comment]
Thank you a lot!.
